src/main/java/org/pentaho/metadata/util/validation/IdValidationUtil.java

```diff
@@ -0,0 +1,67 @@
+/*
+ * This program is free software; you can redistribute it and/or modify it under the
+ * terms of the GNU Lesser General Public License, version 2.1 as published by the Free Software
+ * Foundation.
+ *
+ * You should have received a copy of the GNU Lesser General Public License along with this
+ * program; if not, you can obtain a copy at http://www.gnu.org/licenses/old-licenses/lgpl-2.1.html
+ * or from the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details.
+ *
+ * Copyright (c) 2017 Hitachi Vantara..  All rights reserved.
+ */
+package org.pentaho.metadata.util.validation;
+
+
+import org.pentaho.pms.messages.Messages;
+
+/**
+ * Created by Yury_Bakhmutski on 10/25/2017.
+ */
+public class IdValidationUtil {
```

Comment:
I don't think we should add this class, duplicating the validation logic, since we cannot just remove the methods from the original Util class. 
Let's add this new method to the exisintg Util - you could rename it to be like validateEntityId, leaving those old methods as deprecated. By doing that, you would be able to reuse the validation logic, and in the old method simply convert validation result to boolean (valid/invalid -> true/false).